pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -107,12 +108,12 @@ func runTests(t *testing.T, tests []*validateTest, delta bool) {
 				tt.modify(oc)
 			}
 
-			current := &api.OpenShiftCluster{}
+			var current *api.OpenShiftCluster
 			if delta {
+				current = &api.OpenShiftCluster{}
 				(&openShiftClusterConverter{}).ToInternal(validOpenShiftCluster(), current)
-			} else {
-				(&openShiftClusterConverter{}).ToInternal(oc, current)
```

Comment:
Previously in both `delta == true` and `delta == false` cases we were testing validator as if we were running an update. Now `delta == false` - is a create (`current == nil`) and `delta == true` is an update.

Reply:
GH needs a "good catch" icon...